### ISSUE TITLE
MUX SOP is incorrect

### ISSUE THREAD

**Issue by chukwuemekanweke** [nand2tetris-hdl/nand2tetris-hdl.github.io#1]
The Mux SOP construct is incorrect.

It's supposed to be Ƨa + sb

**Comment by mccallum-sgd** [comment]
Uh maybe that's just a different way of doing it? Is your diagram the same as mine?

**Comment by chukwuemekanweke** [comment]
Yeah the diagram is the same. i'll probably just submit a PR and have you go over it before the end of the week

**Comment by mccallum-sgd** [comment]
Sure sounds good, it's been a while since this class so it's a bit hard to remember. I'll see if I can find my notes.

You're welcome to add to the site btw, there's a lot I didn't get to. I used [Circuit Lab](https://www.circuitlab.com/) for the diagrams.
